mlir/test/Dialect/Vector/test-scalable-bounds.mlir

```diff
@@ -0,0 +1,140 @@
+// RUN: mlir-opt %s -test-affine-reify-value-bounds -cse -verify-diagnostics \
```

Comment:
Can you add a test case that does not contain any `vscale`. Just to make sure everything still works.